config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -72,18 +72,18 @@
 # replicas will not be able to deserialize `BigDecimal` arguments from this
 # serializer. Therefore, this setting should only be enabled after all replicas
 # have been successfully upgraded to Rails 7.1.
-# Rails.application.config.active_job.use_big_decimal_serializer = true
+Rails.application.config.active_job.use_big_decimal_serializer = true
```

Comment:
This is in the category of "needs to have a initial 7.1 deploy before enabling", but is otherwise fine.

Reply:
That's not safe for merging in `main`, we don't have a Rails 7.1 release yet.

Reply:
Ah - I saw that the other marshalling format PR had been merged (which I think also wants a pre-config-change 7.1 deploy to happen first), and thought we were past that threshold. Should I remove the settings with that requirement from this PR? (I think just this one and the marshall metadata one)

Reply:
We sidestepped that for the marshalling format by changing the prefix of all keys in that PR. Yeah, I think it would be best to remove the settings with that requirement from this PR.

Reply:
Turns out it was just this one -- removed it from PR.